### PR TITLE
Intel test/fpga ofs dev 6.1 lts/SRIOV cleanup patches and sync to dev branch

### PR DESCRIPTION
**Pull request by Muralix**

### CODE REVIEW

File: drivers/fpga/dfl-fme-main.c [hunk 1]
```diff
@@ -257,19 +254,19 @@ static int thermal_hwmon_read(struct device *dev, enum hwmon_sensor_types type,
 	switch (attr) {
 	case hwmon_temp_input:
 		v = readq(feature->ioaddr + FME_THERM_RDSENSOR_FMT1);
-		*val = (long)(FIELD_GET(FPGA_TEMPERATURE, v) * 1000);
+		*val = (long)(FIELD_GET(FPGA_TEMPERATURE, v) * MILLI);
```
Comment (rchriste1) [inline drivers/fpga/dfl-fme-main.c:257]: MILLI is not known on RHEL 8.5 and 8.8:
linux-dfl-backport/drivers/fpga/dfl-fme-main.c:257:50: error: ‘MILLI’ undeclared (first use in this function)
   *val = (long)(FIELD_GET(FPGA_TEMPERATURE, v) * MILLI);
                                                  ^~~~~
It compiles and runs on Ubuntu 22.04

**Review by rweight — APPROVED** [review]
Looks good to me

**Review by rchriste1 — CHANGES_REQUESTED** [review]

File: drivers/fpga/dfl-fme-main.c (hunk 1 above)
Comment (Muralix) [reply drivers/fpga/dfl-fme-main.c:257]: defined macro MILLIO in units.h. Compiled on RHEL 8.2.

**Comment by rweight** [comment]
This was merged as a merge-commit instead of a rebase (fast-forward) merge. This was permitted because the rules were never set up properly for this branch. I have fixed the branch-rules and  I have force-pushed over the top to restore a linear history. The code is functionally unchanged. I did, however, split out the units.h changes into a separate backport commit and I changed some spaces to tabs in the units.h file.

**Pull request merged**